libs/wallet/src/web3-react/updater.ts

```diff
@@ -17,8 +17,9 @@ import { gnosisSafeInfoAtom, walletDetailsAtom, walletInfoAtom } from '../api/st
 import { GnosisSafeInfo, WalletDetails, WalletInfo } from '../api/types'
 import { getWalletType } from '../api/utils/getWalletType'
 import { getWalletTypeLabel } from '../api/utils/getWalletTypeLabel'
+import { useSafeAppsSdk } from './hooks/useSafeAppsSdk'
 
-const SAFE_INFO_UPDATE_INTERVAL = ms`30s`
+const SAFE_INFO_UPDATE_INTERVAL = ms`5s`
```

Comment:
This for testing is good, but we will need to reduce it, so Uxio don't get upset

Reply:
Well thought

Reply:
love is in the air :D!